hazelcast-jet-core/src/test/java/com/hazelcast/jet/impl/connector/JmsIntegrationTest.java

```diff
@@ -49,10 +49,13 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import static com.hazelcast.jet.function.DistributedFunctions.noopConsumer;
 import static com.hazelcast.jet.impl.util.Util.uncheckCall;
+import static com.hazelcast.jet.impl.util.Util.uncheckRun;
 import static java.util.Collections.synchronizedList;
 import static java.util.stream.Collectors.toList;
 import static java.util.stream.IntStream.range;
+import static javax.jms.Session.AUTO_ACKNOWLEDGE;
 
 @RunWith(HazelcastParallelClassRunner.class)
 public class JmsIntegrationTest extends JetTestSupport {
```

Comment:
you can consider using `PipelineTestSupport` here instead perhaps?

Reply:
done